components/builder-depot/src/error.rs

```diff
@@ -28,7 +28,10 @@ use redis;
 #[derive(Debug)]
 pub enum Error {
     BadPort(String),
+    ChannelAlreadyExists(String),
+    ChannelDoesNotExist(String),
     DataStore(dbcache::Error),
+    Generic(String),
```

Comment:
You aren't using the `Generic` error anymore, let's remove this and the match arms in the implementations below